packages/ic-response-verification-tests/wasm-tests/http-interface/canister_http_interface.did

```diff
@@ -0,0 +1,47 @@
+// adapted from https://internetcomputer.org/docs/current/references/ic-interface-spec/#http-gateway-interface
```

Comment:
I think it would be good to add a `http-canister` package that encapsulates this interface, like what we have in `ic-utils`. I'll add a ticket for this in our epic on breaking the service worker into smaller packages.